sycl/include/sycl/ext/intel/esimd/memory.hpp

```diff
@@ -148,6 +152,19 @@ gather(const Tx *p, simd<uint32_t, N> offsets, simd_mask<N> mask = 1) {
                                                                  mask.data());
 }
 
+/// A variation of \c gather API with \c offsets represented as \c simd_view
+/// object
+///
```

Comment:
My review request was actually to get rid of description duplication only. Please return parameter doxygen comments.